networkx/classes/graph.py

```diff
@@ -606,67 +606,52 @@ def remove_nodes_from(self, nodes):
             except KeyError:
                 pass
 
-    def nodes_iter(self, data=False):
-        """Return an iterator over the nodes.
+    def nodes(self, data=False):
+        """Returns an iterator over the nodes.
 
         Parameters
         ----------
-        data : boolean, optional (default=False)
-               If False the iterator returns nodes.  If True
-               return a two-tuple of node and node data dictionary
+        data : boolean
```

Comment:
Are we removing the text: optional with default indicated? I would leave this line unchanged.